components/admin-config/backend/src/test/kotlin/AdminConfigIntegrationTest.kt

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (C) 2025 The ORT Server Authors (See <https://github.com/eclipse-apoapsis/ort-server/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.eclipse.apoapsis.ortserver.components.adminconfig
+
+import io.ktor.client.HttpClient
+import io.ktor.server.testing.ApplicationTestBuilder
+
+import org.eclipse.apoapsis.ortserver.shared.ktorutils.AbstractIntegrationTest
+
+@Suppress("UnnecessaryAbstractClass")
+abstract class AdminConfigIntegrationTest(body: AdminConfigIntegrationTest.() -> Unit) : AbstractIntegrationTest({}) {
```

Comment:
Please add a comment for this base class. What is its purpose, for which kind of tests should it be used?